test/jdk/javax/swing/text/DefaultCaret/HidingSelection/HidingSelectionTest.java

```diff
@@ -119,35 +117,12 @@ public static void main(String[] args) throws Exception {
         SwingUtilities.invokeAndWait(anotherWindow::requestFocus);
         robot.waitForIdle();
         robot.delay(200);
-        if (biEqual(robot.createScreenCapture(bounds), nosel)) {
+        if (!field2.getCaret().isSelectionVisible()) {
```

Comment:
Does it need similar change in test/jdk/javax/swing/text/DefaultCaret/HidingSelection/MultiSelectionTest.java
as well?
Since the logic is changed, I will still request to make this test stable on windows (now that we are not using screen capture) and then deproblemlist

Reply:
> Does it need similar change in test/jdk/javax/swing/text/DefaultCaret/HidingSelection/MultiSelectionTest.java as well? Since the logic is changed, I will still request to make this test stable on windows (now that we are not using screen capture) and then deproblemlist

I haven't tested the MultiSelectionTest but since it is problemlisted on Linux similar change might make it more stable. As of this test - since you are the owner of JDK-8194048 i thought that you can review my changes and deproblemlist the test if it is stable on the rest of the platforms and if changes make sense. I definitely do not want to steal your thunder. As of MultiSelectionTest i can take care of it either here or in the original bug that is noted in ProblemList.txt since my fix does not affect it.

Reply:
I guess anyway one is supposed to do a CI job for the fix and test the regression test on all platforms, so if the modified test is problemlisted it will not be tested in CI job, so it's better to deproblemlist and test...
You can do /issue add of that problemlisted bug (it just happens to be in my name for the time being, nothing stopping you assigning to you since you did all the hard work)

Reply:
I have fixed the second test and removed both of them from the problem list.